docs/openapi.yaml

```diff
@@ -350,6 +350,13 @@ paths:
                 # This is my document
 
                 something else here
+          'multipart/form-data':
+            schema:
+              type: object
+              example: |
+                {
+                  "file": [binary file object]
+                }
```

Comment:
If I understand what you're accomplishing here with this, it's that you're using a combination of the `multipart/form-data` content type when used  with a specific field name (`file`) in its payload as a signal that the incoming content is binary data.  I'm guessing you came to this idea because of how old-style HTML forms work for uploading files, but in the modern world there are saner ways of handling this kind of thing -- especially since this itself is a REST API instead of just a bespoke form handling endpoint.

Reply:
I tried set the header, but failed.
I know that you have fixed a error when get binary file, but not **put**.
the code [here](https://github.com/coddingtonbear/obsidian-local-rest-api/blob/main/src/requestHandler.ts#L269)  

only the string type  are available.

i don’t think the multipart/form-data is the key.
The point is that this project did not call the official api `writeBinary `.